website/docs/compaction.md

```diff
@@ -74,7 +74,7 @@ To improve ingestion latency, Async Compaction is the default configuration.
 If immediate read performance of a new commit is important for you, or you want simplicity of not managing separate compaction jobs,
 you may want Synchronous compaction, which means that as a commit is written it is also compacted by the same job.
 
-Compaction is run synchronously by passing the flag "--disable-compaction" (Meaning to disable async compaction scheduling).
+Compaction is run synchronously by passing the flag "--disable-compaction" (Meaning to disable async compaction - disable both scheduling & execution).
 When both ingestion and compaction is running in the same spark context, you can use resource allocation configuration 
```

Comment:
can we just name the param `--disable-async-compaction` to avoid confusion ?

Reply:
+1 on that. I ll file a separate jira for that and keep this PR decoupled for now.